pkg/bootstrap/docker/up.go

```diff
@@ -511,6 +518,22 @@ func (c *ClientStartConfig) EnsureHostDirectories(io.Writer) error {
 	return c.HostHelper().EnsureVolumeShare()
 }
 
+func (c *ClientStartConfig) EnsureDefaultRedirectURIs(out io.Writer) error {
+	f, err := c.Factory()
+	if err != nil {
+		return err
+	}
+
+	patch := fmt.Sprintf("{%q:[%q]}", "redirectURIs", defaultRedirectURI)
+	patchCmd := cmd.NewCmdPatch("", f, out)
+	patchCmd.Flags().Set("patch", patch)
+	patchCmd.Flags().Set("type", "merge")
+	patchArgs := []string{defaultRedirectClient}
+
+	patchCmd.Run(patchCmd, patchArgs)
```

Comment:
@fabianofranz 
If run fails, it won't interrupt the rest of the `oc cluster up` start-up process, as adding a redirect is not critical to the overall function of this command. WDYT?